cmd/test.go

```diff
@@ -123,6 +138,22 @@ func testCmd(ctx *cli.Context) error {
 		return cli.NewExitError("invalid log value "+rawLevel, -1)
 	}
 
+	planFeatures := models.FeatureMap{}
+	if sPlanFeatures != "" {
+		err := json.NewDecoder(strings.NewReader(sPlanFeatures)).Decode(planFeatures)
```

Comment:
You can also use `json.Unmarshal` since you already have the string/[]byte in memory.

Reply:
ooh I like that better :)